src/app/constants/flags/gameobject-flags.ts

```diff
@@ -0,0 +1,14 @@
+import { Flag } from '../../types/general';
+
+export const GAMEOBJECT_FLAGS: Flag[] = [
+  { bit: 0,   name: 'NONE'           },
```

Comment:
`IN_USE` should start from bit 0 (because its value is 1, so 2^0 = 1) and remove `name: 'NONE'  `